Components/ValidationService.php

```diff
@@ -378,14 +378,16 @@ private function sendShopOwnerEmail(VatIdCustomerInformation $customerInformatio
      */
     private function getEmailAddress()
     {
-        $emailNotification = $this->config->get(VatIdConfigReaderInterface::EMAIL_NOTIFICATION);
+        $emailNotification = (string) $this->config->get(VatIdConfigReaderInterface::EMAIL_NOTIFICATION);
```

Comment:
maybe you cast it to boolean

Reply:
the plugin offers the possibility to choose between 1 and 0 AND an email address. you can see it in the test. Di you still think a boolean would be the better choice?